test/views/UploadYourFilesConfirmationSpec.scala

```diff
@@ -74,10 +74,25 @@ class UploadYourFilesConfirmationSpec extends DomAssertions with ViewMatchers wi
       }
     }
 
-    "have a links to print the page" in {
+    "have a link to print the page" in {
       assertContainsLink(view, messages(s"$pagePrefix.printPage"), "javascript:if(window.print)window.print()")
     }
 
+    "have a paragraph explaining what happens when a query is raised" which {
+      "has a heading" in {
+        view.getElementsByTag("h3").first() must containMessage(s"$pagePrefix.section1.title")
```

Comment:
Even if it's working as it is now (is it?), I would change it to `${pagePrefix.section1.title}` ?

Reply:
I think you are missing the contet here. `pagePrefix` is a field defined at the top of this file.
But it confused me as well, so I'll change it so there is no misunderstanding anymore.